inf-clojure.el

```diff
@@ -807,22 +810,24 @@ process buffer for a list of commands.)"
                                           nil
                                           'confirm-after-completion))))
   (let* ((project-dir (clojure-project-dir))
-         (process-buffer-name (if project-dir
-                                  (format "inf-clojure %s" (inf-clojure--project-name project-dir))
-                                "inf-clojure"))
+         (process-buffer-name (or
+                               inf-clojure-custom-repl-name
+                               (if project-dir
+                                   (format "inf-clojure %s" (inf-clojure--project-name project-dir))
+                                 "inf-clojure")))
          ;; comint adds the asterisks to both sides
-         (repl-buffer-name (format "*%s*" process-buffer-name)))
+         (repl-buffer-name (format "*%s*" process-buffer-name))
+         (repl-type (or (unless prefix-arg
```

Comment:
why was `repl-type` moved up? Is there a functional change there or just moving it up in the lexical scope?